CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # ezcater_rubocop
 
+## v0.51.1
+- Disable `Rails/FilePath` cop.
+- Disable `Style/EmptyLiteral` cop.
+
+## v0.51.0
```

Comment:
Ah sorry for not doing this - I'll be sure to next time. Thanks for taking care of this one.

Reply:
NP - your reviewer should have caught it :)